modules/ROOT/pages/get-started.adoc

```diff
@@ -0,0 +1,143 @@
+= Getting Started with Partner Manager
+
+Partner Manager offers a simplified, configuration-driven B2B integration solution to manage B2B trading partner ecosystems. To use Partner Manager, follow these steps:
+
+. <<assign-permissions,Assign permissions to Partner Manager users>>.
+. <<set-up-env,Set up the Partner Manager environment>>.
+. <<payload-storage-api,Set up the Payload Storage API>>.
+. <<configure-host-profile,Configure a host profile>>.
+. <<configure-partners,Configure a profile for each of your trading partners>>.
+. <<configure-third-party,Optionally configure third party connections>>.
+. <<configure-message-flows,Configure message flows>>.
+. <<deploy-test-message-flows,Deploy and test the configured message flows>>.
+. <<monitor-transmissions,Monitor transmissions and individual messages>>.
+. <<troubleshoot-acks,Troubleshoot rejected or overdue acknowledgements>>.
+
+[[assign-permissions]]
+== Assign Permissions to Partner Manager Users
+
+For each sandbox and production environment, use Anypoint Access Management to assign permissions to each Partner Manager user. You can assign the following permissions:
+
+*  *Partner Manager Administrator* 
++
+** Has full permissions
+* *View Host, Partners, and Message Flows* 
++
+** Has view-only access to the host, partner, and message flow configurations
+** Cannot view transaction activity
+* *Manage Partners and Message Flows* 
++
+** Can create, modify, and delete partners or message flow configurations
+** Can view host partner configurations
+** Cannot view and manage transaction activity
+* *Manage Activity* 
++
+** Can view and manage transaction activity
+** Cannot view or modify either partner or message flow configurations
+* *Manage Host* 
++
+** Can create, modify, and delete host configurations
+** Cannot view or modify trading partner configurations or transaction activity 
+
++
+This access applies even if the user has the Anypoint Platform Org Admin role.
+* *View Activity* 
++
+** Has view-only access to transaction activity
+** Cannot view or modify either partner or message flow configurations
+
+For more information, see xref:access-management::permissions-by-product.adoc[] .
+
+[[set-up-env]]
+== Set Up the Partner Manager Environment
+
+You can deploy Partner Manager to CloudHub or you can use Partner Manager on-premises:
+
+* To deploy Partner Manager to CloudHub, use the *Partner Manager Settings* page in Anypoint Access Management to specify CloudHub deployment options for the Partner Manager sandbox and production environments.
```

Comment:
I think Kim advised against "using a page" to do something, but I can't remember. 

It might be good to be either more specific (the path to that option: Access Management > ?? > Partner Manager) or less specific (configure PM settings in Access Management). 

It seems weird to use AIM to deploy to CH, tho. I would expect RTM. 

Also, you might want to specify which CloudHub (CH1 or CH2 or both).

Reply:
Rewritten